salt/metalk8s/orchestrate/worker_plane.sls

```diff
@@ -0,0 +1,93 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
```

Comment:
Since everything here is on the target node and a 'state', over time this `playbook` should basically only invoke `state.highstate` on the target node, given we have an appropriate 'top' generated by a `master_tops` module.

Reply:
Agree